migration-util/src/main/java/edu/unc/lib/dcr/migration/content/ACLTransformationHelpers.java

```diff
@@ -72,9 +75,9 @@ private ACLTransformationHelpers() {
      */
     public static void transformPatronAccess(Resource bxc3Resc, Resource bxc5Resc, PID parentPid) {
         // For admin units, cache patron access settings so they can be used for children instead
+        Model unitModel = createDefaultModel();
```

Comment:
you shouldn't need to create a new model to create a literal here, since you have existing models. You can do `bxc3Resc.getModel()` or `bxc5Resc.getModel()`. There's also a static method for creating typed literals, `ResourceFactory.createTypedLiteral()` which would probably work, although I haven't tried it